### PR TITLE
PLAT-612: method call when constructor execution is pending

### PR DESCRIPTION
**Pull request by MariaVasilenko**
**- What I did**
- added test with skip, because logic is not implemented
- updated testrail [case](https://insolar.testrail.io/index.php?/cases/view/5237&group_by=cases:section_id&group_order=asc&group_id=943) from 2 to 1

### CODE REVIEW

File: ledger-core/virtual/integration/method_pulse_change_test.go [hunk 1]
```diff
@@ -419,3 +419,135 @@ func TestVirtual_Method_CheckPendingsCount(t *testing.T) {
...
+			)
+
+			server.IncrementPulseAndWaitIdle(ctx)
+			p2 := server.GetPulse().PulseNumber
+
+			// create object state
+			{
+				payload := &payload.VStateReport{
+					Status:                      payload.Empty,
+					Object:                      object,
```
Comment (ruz) [inline ledger-core/virtual/integration/method_pulse_change_test.go:474]: Add AsOf field equal to p1

File: ledger-core/virtual/integration/method_pulse_change_test.go [hunk 2]
```diff
@@ -419,3 +419,135 @@ func TestVirtual_Method_CheckPendingsCount(t *testing.T) {
...
+					OrderedPendingCount:         1,
+					OrderedPendingEarliestPulse: p1,
+				}
+
+				server.WaitIdleConveyor()
```
Comment (ruz) [inline ledger-core/virtual/integration/method_pulse_change_test.go:479]: no need in above wait

File: ledger-core/virtual/integration/method_pulse_change_test.go [hunk 3]
```diff
@@ -419,3 +419,135 @@ func TestVirtual_Method_CheckPendingsCount(t *testing.T) {
...
+				requestResult := requestresult.New([]byte("call result"), gen.UniqueGlobalRef())
+
+				objectExecutionMock := runnerMock.AddExecutionMock("SomeMethod")
+				objectExecutionMock.AddStart(
+					func(ctx execution.Context) {
+						logger.Debug("ExecutionStart [SomeMethod]")
+						require.Equal(t, object, ctx.Request.Callee)
+						// check new obj state (after completed constructor call)
+						require.True(t, getDelegated)
+						require.Equal(t, []byte("new object memory"), ctx.Request.Arguments) // todo
```
Comment (ruz) [inline ledger-core/virtual/integration/method_pulse_change_test.go:496]: Request.Arguments is not object's memory

File: ledger-core/virtual/integration/method_pulse_change_test.go [hunk 4]
```diff
@@ -419,3 +419,135 @@ func TestVirtual_Method_CheckPendingsCount(t *testing.T) {
...
+				Caller:              server.GlobalCaller(),
+				Callee:              object,
+				CallSiteDeclaration: gen.UniqueGlobalRef(),
+				CallSiteMethod:      "SomeMethod",
+				CallOutgoing:        server.BuildRandomOutgoingWithPulse(),
+			}
+
+			server.SendPayload(ctx, &pl)
+
+			finished := payload.VDelegatedRequestFinished{
```
Comment (ruz) [inline ledger-core/virtual/integration/method_pulse_change_test.go:532]: I don't see delegation asked with a message.

File: ledger-core/virtual/integration/method_pulse_change_test.go (hunk 1 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/method_pulse_change_test.go:474]: done

File: ledger-core/virtual/integration/method_pulse_change_test.go (hunk 2 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/method_pulse_change_test.go:479]: done

File: ledger-core/virtual/integration/method_pulse_change_test.go (hunk 4 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/method_pulse_change_test.go]: done

**Pull request merged**